src/client/src/app/site/[site]/view-availability/week/view-week-availability-page.tsx

```diff
@@ -1,35 +1,99 @@
-import { Card, Table } from '@components/nhsuk-frontend';
-import { DailyAvailability } from '@types';
+import { Card, Pagination, Table } from '@components/nhsuk-frontend';
+import { DayAvailabilityDetails } from '@types';
 import dayjs from 'dayjs';
 
 type Props = {
-  availability: DailyAvailability[];
+  days: DayAvailabilityDetails[];
+  weekStart: dayjs.Dayjs;
+  weekEnd: dayjs.Dayjs;
 };
 
-export const ViewWeekAvailabilityPage = ({ availability }: Props) => {
+export const ViewWeekAvailabilityPage = ({
+  days,
+  weekStart,
+  weekEnd,
+}: Props) => {
+  const nextWeek = weekStart.add(1, 'week');
+  const previousWeek = weekStart.add(-1, 'week');
+
+  let previousWeekDisplay = '';
+  let nextWeekDisplay = '';
+
+  if (previousWeek.month() < weekStart.month()) {
+    previousWeekDisplay = `${previousWeek.format('D MMMM')}-${weekEnd.add(-1, 'week').format('D MMMM YYYY')}`;
+  } else {
+    previousWeekDisplay = `${weekStart.add(-1, 'week').format('D')}-${weekEnd.add(-1, 'week').format('D MMMM YYYY')}`;
```

Comment:
Why do we not use previousWeek in place of weekStart.add(-1 'week')? This is also confusing so maybe refactor to a method so that we can document what this is doing in the method name

Reply:
Yeah I should have used previous week for both, I'll clean this up